src/lib/bcmp/bcmp.h

```diff
@@ -3,6 +3,9 @@
 #include <stdbool.h>
 #include <stdint.h>
 
+extern "C" {
+#include "device.h"
+}
```

Comment:
For the time being, until all of the messages files are c-ified, C++ files need to know how to link the current C source files. Since the whole library for `bm_core` will be in C, I figured we do not want to link "private" api to `bm_core` with the possibility of there being C++ files ex:
## Some 'bm_core` header file:
```c
#ifdef __cplusplus
extern "C" {
#endif

...

#ifdef __cplusplus
}
#endif
```

Reply:
Ah, got it. Thank you. Very clear. 👍